web3-dev/smart-contracts/deploy-from-local-node.rst

```diff
@@ -0,0 +1,86 @@
+.. index:: smart contracts, assemblyscript
+
+.. _deploy-from-local-node:
+
+Deploy a smart contract on a local node
+========================================
+
+In this section we will execute a smart contract by running your own node.
+
+To execute the smart contract you will need:
+
+- A client configured with an address having coins.
+- A smart contract compiled in WebAssembly. We will use the `hello world` contract from the :ref:`getting-started <sc-getting-started>` step.
+
+.. note::
+   * If you don't have any wallet configured yet, :ref:`create a new one <wallet>`.
+   * If you're using a brand new wallet, add some coins by sending your address to
+     `testnet-faucet discord channel <https://discord.com/channels/828270821042159636/866190913030193172>`_.
+   * In any case, keep the `Address` and `Secret key` of your wallet, you will use it later.
+
+Let's go!
+
+Configure the client
+~~~~~~~~~~~~~~~~~~~~
+
+Make sure that you have the latest version of the Massa node. If not,
+:ref:`install it <testnet-install>` and :ref:`run it <testnet-running>`.
+
+.. note::
+
+   You can also execute your smart contract on a local sandbox node.
```

Comment:
here we should only talk about local node I think because the title of that section is "Deploy a smart contract on a local node"

Reply:
Yes, but then we should probably have a section that is not specific to local node only. @qdrn @Thykof what do you think?